app/modules/users/schemas.py

```diff
@@ -94,10 +94,13 @@ class PersonalUserSchema(DetailedUserSchema):
     that can be edited by the currently logged in user.
     """
 
+    collaborations = base_fields.Function(User.get_collaborations_as_json)
+
     class Meta(DetailedUserSchema.Meta):
         fields = DetailedUserSchema.Meta.fields + (
             User.unprocessed_asset_groups.__name__,
             User.unprocessed_sightings.__name__,
+            'collaborations',
```

Comment:
Have a think about this, do we really want the full details for all collaborations each time the user conects?

Reply:
I think it's okay and can be changed later if it becomes too expensive. I imagine on most platforms users will only have a handful of collaborations.